### ISSUE TITLE
Provide an encoder implementation in Java

### ISSUE THREAD

**Issue by deshanxiao** [google/brotli#1060]
Is it possible for us to support the java version of Brotli's encoder?

**Comment by eustas** [comment]
Hello. Do you mean pure Java encoder?

**Comment by deshanxiao** [comment]
Hi @eustas yes, I try to find java encoder in https://mvnrepository.com/artifact/org.brotli. But it seems that there is just a `dec `  component.

**Comment by deshanxiao** [comment]
In fact, it can also be accpeted in the form of JNI if we could support enough system architecture (like aarch64).

Since there is no official java encoder support, we have to use a library such as https://github.com/hyperxpro/Brotli4j

> What quality range (fast, balanced, maximal density) is the primary target? Knowing that would allow us to focus on it first.

It might be better if users can adjust these parameters according to their needs.

**Comment by eustas** [comment]
Agree. We could do that much faster (and encoder will be much more performant).
I'll update issue title then.

For pure java encoder there is #405

**Comment by deshanxiao** [comment]
Thank you @eustas! If you don't mind please let me know if there is any progress.

**Comment by deshanxiao** [comment]
Hi @eustas Do we have any plans to implement this feature for Java users?
